Fearless Concurrency/Using Message Passing to Transfer Data Between Threads/Sum All Numbers/task.md

```diff
@@ -0,0 +1,11 @@
+## Sum All Numbers
+
+We want to send elements of two vectors to a channel, and then count and find a sum of all of them.
```

Comment:
We want to send the elements of two vectors to a channel and then count and find the sum of all of them.